Vimeo-Native/VimeoInterop/VimeoInterop.vcxproj.filters

```diff
@@ -24,6 +24,9 @@
     <ClInclude Include="resource.h">
       <Filter>Header Files</Filter>
     </ClInclude>
+    <ClInclude Include="..\Include\Vimeo\VimeoAPI.h">
```

Comment:
Yea, I kinda figured for the most part, but I just wanted to make sure that it was necessary. @bberton can probably tell better than I.